dashboard/src/main/java/org/openedx/courses/presentation/DashboardGalleryViewModel.kt

```diff
@@ -145,6 +148,7 @@ class DashboardGalleryViewModel(
         }
         _updating.value = isUpdating
         getCourses(isIAPFlow = isIAPFlow)
+        viewModelScope.launch { pushNotifier.send(PushEvent.RefreshPushEvent) }
```

Comment:
This will affect the atomic-ness of the method, it is better to create a dedicated function for this and call it from the same place we are calling this `updateCourses` method.

Reply:
dedicated method created.